server/assets.py

```diff
@@ -47,6 +47,7 @@
 
 student_css = Bundle(
     'css/student.css',
+    'css/highlight.css',
```

Comment:
I outlined the parts in yellow. There's some extra padding that applies to the sidebar and a bit of the navbar. 

![alignment](https://cloud.githubusercontent.com/assets/882381/12798399/1208d7ce-ca7e-11e5-8c91-b16efd83fa09.png)